stat/distmat/wishart.go

```diff
@@ -145,19 +141,19 @@ func (w *Wishart) logProbSymChol(cholX *mat.Cholesky) float64 {
 	return 0.5*((fnu-fdim-1)*logdetx-tr-fnu*fdim*math.Ln2-fnu*w.logdetv) - mathext.MvLgamma(0.5*fnu, w.dim)
 }
 
-// RandSym generates a random symmetric matrix from the distribution.
-func (w *Wishart) RandSym(x *mat.SymDense) *mat.SymDense {
-	if x == nil {
-		x = &mat.SymDense{}
-	}
+// RandSymTo generates a random symmetric matrix from the distribution.
+// If dst is empty, it is resized to be an d×d symmetric matrix where d is the order
+// of the receiver. When dst is non-empty, RandSymTo panics if dst is not d×d.
+func (w *Wishart) RandSymTo(dst *mat.SymDense) {
 	var c mat.Cholesky
-	w.RandChol(&c)
-	c.ToSym(x)
-	return x
+	w.RandCholTo(&c)
+	c.ToSym(dst)
 }
 
-// RandChol generates the Cholesky decomposition of a random matrix from the distribution.
-func (w *Wishart) RandChol(c *mat.Cholesky) *mat.Cholesky {
+// RandCholTo generates the Cholesky decomposition of a random matrix from the distribution.
+func (w *Wishart) RandCholTo(dst *mat.Cholesky) {
+	// TODO(kortschak): Make RandCholTo care about the size of dst.
+
 	// TODO(btracey): Modify the code if the underlying data from c is exposed
```

Comment:
s/`c`/`dst`

Reply:
Should I change this now, or when I remove the TODO above with the change at Cholesky?